### ISSUE TITLE
Merge the markup from atomic/blocks/product-elements/ with base/components/cart-checkout/

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce#42680]
Initially discussed in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1959#discussion_r391842881.

We have very similar components in `atomic/blocks/product-elements/` (used for the _All Products_ and _Single Product_ blocks) and `base/components/cart-checkout/` (used for the _Cart_ and _Checkout_ blocks). This issue is about moving the markup from Atomic blocks to independent components as much as possible. Probably that would mean moving some components out of the `cart-checkout` subfolder.

**Comment by WunderBart** [comment]
👋 @Aljullu is this issue still relevant? I'm trying to make sure it's in appropriate backlogs 🙌

**Comment by Aljullu** [comment]
Yes, I think we can close it. As we are moving to PHP-rendered blocks for everything under `atomic/blocks/product-elements/`, I don't think it's worth investing into refactoring the JS-side of those blocks. Thanks for the heads-up, @WunderBart!
